examples/specialty_plots/swotplot.py

```diff
@@ -0,0 +1,180 @@
+"""
+============
+SWOT Diagram
+============
+
+SWOT analysis (or SWOT matrix) is a strategic planning and strategic management
+technique used to help a person or organization identify Strengths, Weaknesses,
+Opportunities, and Threats related to business competition or project planning.
+It is sometimes called situational assessment or situational analysis.
+Additional acronyms using the same components include TOWS and WOTS-UP.
+Source: https://en.wikipedia.org/wiki/SWOT_analysis
+
+"""
+import logging
+from typing import Tuple
+
+import matplotlib.pyplot as plt
+
+_log = logging.getLogger(__name__)
+
+
+def draw_quadrant(q_data: dict, ax: plt.Axes, q_color: str,
+                  q_x: Tuple[float, float, float, float],
+                  q_y: Tuple[float, float, float, float],
+                  q_title_pos: Tuple[float, float] = (0.0, 0.0),
+                  q_title_props: dict = dict(boxstyle='round',
+                                             facecolor="white",
+                                             alpha=1.0),
+                  q_point_props: dict = dict(boxstyle='round',
+                                             facecolor="white",
+                                             alpha=1.0)
+                  ) -> None:
+    """
+    Draw a quadrant of the SWOT plot
+
+    Parameters
+    ----------
+    q_data : dict
+        Data structure passed to populate the quadrant.
+        Format 'key':Tuple(float,float)
+    ax : plt.Axes
+        Matplotlib current axes
+    q_color : str
+        Quadrant color.
+    q_x : Tuple[float]
+        Quadrant X relative coordinates. Format: Tuple(float,float,float,float)
+    q_y : Tuple[float]
+        Quadrant Y relative coordinates. Format: Tuple(float,float,float,float)
```

Comment:
Shouldn't the format go with the type?
```suggestion
    q_x : Tuple[float, float, float, float]
        Quadrant X relative coordinates.
    q_y : Tuple[float, float, float, float]
        Quadrant Y relative coordinates.
```
I also don't really understand what relative means; relative to what?

Reply:
Removing "relative" adjective, the plot is still clipped between 0 and 1 both in x and y axes.

Reply:
Please see comment about JSON formatting below